crates/ironrdp-web/src/clipboard/transaction.rs

```diff
@@ -19,7 +19,7 @@ impl ClipboardTransaction {
 
 #[wasm_bindgen]
 impl ClipboardTransaction {
-    pub fn new() -> Self {
+    pub fn construct() -> Self {
```

Comment:
question: Is there a reason for not using `new` here?

Reply:
We can't have `new` in an interface

Reply:
Fixed in https://github.com/Devolutions/IronRDP/pull/747